src/gr.h

```diff
@@ -1404,6 +1405,12 @@ void gr_ctx_init_gr_mpoly(gr_ctx_t ctx, gr_ctx_t base_ring, slong nvars, const o
 void gr_ctx_init_fmpz_mpoly_q(gr_ctx_t ctx, slong nvars, const ordering_t ord);
 #endif
 
+/* Generic Ore polynomials */
```

Comment:
I don't think there needs to be a constructor for this ring in ``gr.h``.

It suffices to have ``gr_ore_poly_ctx_init`` in ``gr_ore_poly.h``.